pywatts/modules/max_error.py

```diff
@@ -0,0 +1,101 @@
+import logging
+from typing import Dict, Optional
+
+import xarray as xr
+import numpy as np
+import pandas as pd
+
+from pywatts.core.base import BaseTransformer
+from pywatts.core.exceptions.input_not_available import InputNotAvailable
+from pywatts.utils._xarray_time_series_utils import _get_time_indeces
+
+logger = logging.getLogger(__name__)
+
+
+class MaxErrCalculator(BaseTransformer):
```

Comment:
Due to Summary PR #85, RMSECalculator module will be renamed just to RMSE. So, should we use Min/Max or MinErr/MaxErr instead of MinErrCalculator/MaxErrCalculator?